kcidb/test_oo.py

```diff
@@ -0,0 +1,101 @@
+"""kcidb.oo test module"""
+
+import os
+import kcidb
+import kcidb.db as db
+import kcidb.oo as oo
+
+from kcidb.test_orm import parse
```

Comment:
The `parse()` in `kcidb.test_orm` is just a tiny convenience function. Let's not create unnecessary dependencies, especially between tests, which are often ephemeral. Just copy `parse()` over here, if you need it (it seems you don't right now).

Please note, though, that `parse()` in `kcidb.test_orm` is using its own schema, and not **the** schema defined in `kcidb.orm.SCHEMA` and used by `kcidb.oo`. So it won't work properly for you.

Reply:
I wanted to use it, but I ended up not using it. 
I forgot to remove the import.